### PR TITLE
Sportschau added

### PR DESCRIPTION
**Pull request by JuSteif**
Sportschau is a german sports magazine

### CODE REVIEW

File: src/fundus/publishers/de/__init__.py [hunk 1]
```diff
@@ -30,6 +31,17 @@
...
 class DE(PublisherEnum):
+    SportSchau = PublisherSpec(
+        name="Sportschau",
+        domain="https://www.sportschau.de/",
+        parser=SportSchauParser,
+        sources=[
+            RSSFeed("https://www.sportschau.de/index~rss2.xml"),
+            Sitemap("https://www.sportschau.de/index~sitemap_p-0.xml"),
+            NewsMap("https://www.sportschau.de/kompakt-sp-100~news.xml"),
+        ],
```
Comment (addie9800) [inline src/fundus/publishers/de/__init__.py:42]: For some reason sportschau seems to also link articles from other websites such as mdr, which then naturally can't be parsed by your Parser. I would suggest adding `url_filter=inverse(regex_filter("sportschau.de"))` to your PublisherSpec. Don't worry if it doesn't seem to make a difference yet, this is due to a bug that is fixed with #459

File: src/fundus/publishers/de/sportschau.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class SportSchauParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("article >p.textabsatz")
```
Comment (addie9800) [inline src/fundus/publishers/de/sportschau.py:16]: As of now, this selector also include the summary. I would suggest extending this selector by `article >p.textabsatz:not(:nth-of-type(1))`

**Review by addie9800 — CHANGES_REQUESTED** [review]
Thanks a lot for adding this 👍

File: src/fundus/publishers/de/__init__.py (hunk 1 above)
Comment (JuSteif) [reply src/fundus/publishers/de/__init__.py:42]: sorry, what does "inverse" do, because it seems like it does not work

Comment (addie9800) [reply src/fundus/publishers/de/__init__.py:42]: Inverse works like a logical not for filters. I did forget to mention you still have to import it though. Sorry about that:

`from fundus.scraping.filter import inverse, regex_filter`

Comment (addie9800) [reply src/fundus/publishers/de/__init__.py:42]: The PR looks good, I just forgot to mention that you have to regenerate the test files if you update the extraction logic: `scripts.generate_parser_test_files -p SportSchau -oj`

**Review by MaxDall — DISMISSED** [review]

**Review by addie9800 — APPROVED** [review]

**Pull request merged**